.circleci/config.yml

```diff
@@ -157,28 +179,75 @@ workflows:
     jobs:
       - trigger-workflows
 
+  # Trigger tag build when tag and not already building a specific package workflow
+  tag:
+    when: 
+      and:
+        - << pipeline.git.tag >>
+        - not: << pipeline.parameters.wpe-headless-plugin >>
+    jobs:
+      - trigger-tag:
+          filters:
+            tags:
+              only: /^plugin\/wpe-headless\/\S+/
+
   # Workflows defined for each package.
   plugin-wpe-headless:
     when: << pipeline.parameters.wpe-headless-plugin >>
     jobs:
-      - checkout
+      - checkout:
+          filters:
+            tags:
+              only: /.*/
       - plugin-wpe-headless-checkout:
           requires:
             - checkout
+          filters:
+            tags:
+              only: /.*/
       - plugin-wpe-headless-composer:
           requires:
             - plugin-wpe-headless-checkout
+          filters:
+            tags:
+              only: /.*/
       - plugin-wpe-headless-bundle-zip:
           requires:
             - plugin-wpe-headless-checkout
+          filters:
+            tags:
+              only: /.*/
       - plugin-wpe-headless-bundle-json:
           requires:
             - plugin-wpe-headless-checkout
-      - plugin-wpe-headless-deploy-staging:
+          filters:
+            tags:
+              only: /.*/
+      - plugin-wpe-headless-deploy:
+          name: "Deploy zip to api (staging)"
+          requires:
+            - plugin-wpe-headless-bundle-zip
+            - plugin-wpe-headless-bundle-json
+          filters:
+            branches:
+              only:
+                - main
```

Comment:
@rfmeier asked if this should be canary merge.  It could be both. :looking: at that now.

Reply:
Yeah, I think we'll want both `canary` and `main` here.